gatorgrade/generate/generate.py

```diff
@@ -49,6 +51,7 @@ def create_targeted_paths_list(
             if filename.startswith("__") or filename.startswith("."):
                 continue
             # Combine the path with file name to get a complete path
+
```

Comment:
Is this space needed? I think it should at least be above the comment, so that there is no space between the comment describing line 55 and line 55 itself.

Reply:
This blank line is created by `black` automatically. I'd like to know its purpose as well. Lol

Reply:
You may be able to manually move the comment and keep a blank line above that, then black might be okay with it.